packages/jaeger-ui/src/components/TracePage/index.tsx

```diff
@@ -376,37 +377,45 @@ export class TracePageImpl extends React.PureComponent<TProps, TState> {
     };
 
     return (
-      <div>
-        {archiveEnabled && (
-          <ArchiveNotifier acknowledge={this.acknowledgeArchive} archivedState={archiveTraceState} />
-        )}
-        <div className="Tracepage--headerSection" ref={this.setHeaderHeight}>
-          <TracePageHeader {...headerProps} />
+      <ThemeProvider
+        value={
+          {
```

Comment:
As mentioned [here](https://github.com/jaegertracing/jaeger-ui/pull/520#discussion_r387282602), this should be a `const` at the top of the file otherwise `ThemeProvider` consumers will re-render on every render of `TracePage/index`